examples/generate-python-models/__snapshots__/index.spec.ts.snap

```diff
@@ -0,0 +1,18 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Should be able to render python models and should log expected output to console 1`] = `
+Array [
+  "class Root: 
```

Comment:
I dont get why this is created 😆 Cause nothing of these changes should affect it 🧐 

Oh well, must be a missed when the PR was merged.

Reply:
Probably 😄